src/mqtt/qmqtt_global.h

```diff
@@ -34,14 +34,12 @@
 
 #include <QtGlobal>
 
-#ifndef QT_STATIC
-#  if defined(QT_BUILD_QMQTT_LIB)
-#    define Q_MQTT_EXPORT Q_DECL_EXPORT
-#  else
-#    define Q_MQTT_EXPORT Q_DECL_IMPORT
-#  endif
-#else
+#if defined(QT_BUILD_QMQTT_LIB)
```

Comment:
This change could break things. The QT_STATIC macro is defined when compiling with a static QT build. Because the qmake build files are made to create a QT module, qmqtt should also be build as a static library.

I would propose to restore the original code here and define a new macro in the new qmqtt.pri file to indicate that we're not trying to build a QT module.

Next replace `ifndef QT_STATIC` with `if !defined QT_STATIC && !defined YOUR_NEW_MACRO`